kubectl-rook-ceph.sh

```diff
@@ -263,6 +265,83 @@ function run_purge_osd() {
     rook ceph osd remove --osd-ids=$1 --force-osd-removal=$force_removal"
 }
 
+####################################################################################################
+# 'kubectl rook-ceph debug commands
+####################################################################################################
+
+function update_deployment_spec_command(){
+  deployment_spec=$1
+  main_command="[\"sleep\",\"infinity\"]"
+  deployment_spec=$(jq .template.spec.containers[0].command="$main_command" <<< $deployment_spec)
+  deployment_spec=$(jq .template.spec.containers[0].args=[] <<< $deployment_spec)
+}  
+
+function update_deployment_spec_image(){
+  deployment_spec=$1
+  alternalte_image=$2
+  deployment_spec=$(jq .template.spec.containers[0].env[0].value="$alternalte_image" <<< $deployment_spec)
+  deployment_spec=$(jq .template.spec.containers[0].image="$alternalte_image" <<< $deployment_spec)
+}
+
+function run_create_debug(){
+  [[ -z "${1:-""}" ]] && fail_error "Missing 'mon/osd/mgr/mds' deployment name"
```

Comment:
Do we need to specify mon/osd/mgr/mds here? Do we support mgr/mds at this point in time?

```suggestion
  [[ -z "${1:-""}" ]] && fail_error "Missing deployment name"
```

Reply:
I think we should only support the debug pods for mons and osds. Everything else should be doable in the toolbox. So maybe we can add an error check that it's only for mons and osds
```suggestion
  [[ -z "${1:-""}" ]] && fail_error "Missing mon or OSD deployment name"
```